scalingo/provider.go

```diff
@@ -17,12 +19,22 @@ func Provider() terraform.ResourceProvider {
 			"api_url": {
 				Type:        schema.TypeString,
 				Optional:    true,
-				DefaultFunc: schema.EnvDefaultFunc("SCALINGO_API_URL", "https://api.scalingo.com/"),
+				DefaultFunc: schema.EnvDefaultFunc("SCALINGO_API_URL", nil),
+			},
+			"db_api_url": {
```

Comment:
Is it really needed to specify the DB API URL and the API URL? Just the region is enough, isn't it?

Reply:
For the production public environment no, but for dev/staging/private deployments, yes :)